code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -88,7 +125,12 @@
 			world << pick(creepyasssounds)
 		else
 			lastsinPerson = world.time
-			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			var/mob/living/carbon/human/sinPerson = pick(living_mob_list)
+			var/depth = living_mob_list.len + 1 // just in case

```

Comment:
Oh, I was thinking & returned both lists combined, didn't know that sort of operator worked for list. I'll add that in the next update I do for the bishop.